documentation/modules/proc-downloading-product.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-product-downloads-{context}']
+= Downloading {ProductName}
+
+Download a ZIP file that contains the resources required for installation along with examples for configuration.
+
+.Prerequisites
+
+* Access to {ReleaseDownload}
+
+.Procedure
+
+. Download the `strimzi-x.y.z.zip` file from {ReleaseDownload}.
```

Comment:
Should x y and z be italic to highlight that they're replaceable?

Reply:
We don't currently do that. I think we followed a convention used with other products. Italics is used for info the user is meant to replace.